cancat/uds/utils.py

```diff
@@ -0,0 +1,533 @@
+# Utility functions for UDS
+
+import time
+import string
+import struct
+from contextlib import contextmanager
+
+from cancat import uds
+from cancat.uds import UDS
+from cancat.utils import log
+from cancat.utils.types import ECUAddress, _range_func
+
+
+def get_uds_29bit_srcid(arbid):
```

Comment:
i'm not familiar with uds in 29bit light-duty.  the only application of standard meaning to ARBID is in J1939 afaik.  is this misidentified?